experimental/css-has-pseudo/src/index.js

```diff
@@ -131,10 +131,39 @@ function selectorSpecificity(node) {
 			case ':has':
 			case ':not':
 				{
-					const pseudoSpecificity = selectorSpecificity(node.nodes[0]);
-					a += pseudoSpecificity.a;
-					b += pseudoSpecificity.b;
-					c += pseudoSpecificity.c;
+					if (node.nodes && node.nodes.length > 0) {
+						let mostSpecificListItem = {
+							a: 0,
+							b: 0,
+							c: 0,
+						};
+
+						node.nodes.forEach((child) => {
+							const itemSpecificity = selectorSpecificity(child);
```

Comment:
This is why I was mentioning extracting since I've been seeing this

Reply:
Yes, each time I copy/paste a bit I remind myself that I should fine a way to have these imported from a central util bundle :D